src/components/Selector.js

```diff
@@ -20,13 +20,13 @@ export class RawSelector extends React.Component {
       <select
         className={this.props.cssClasses.select}
         onChange={this.handleChange}
-        value={currentValue}
+        value={`${currentValue}`}
```

Comment:
Why did you set them to string? If we keep value as number, it's still works. The coercion is made by the browser.

Reply:
I will try then. :)

Reply:
Oh know, I have to transform to string because of `undefined`...

Reply:
Yes you are right... Maybe at some point we should also support empty string as reset value. With it we don't need to cast number to string in the template, just provide the default value as `""`.

Reply:
Well I think the undefined is the most natural move since you can completely avoid the `value` and I don't think we should support more than one "0" value. However we could change it to `""` if you and the team think it's best.